### ISSUE TITLE
The project was not configured: run "waf configure" first!

### ISSUE THREAD

**Issue by memegauste** [CrashCortez/vitadock-mpv#4]
```Installing with ./waf install...

========================= Installation results ===========================
The project was not configured: run "waf configure" first!

****  Installation failed. Aborting package creation.

Cleaning up...OK

Bye.

vidware: line 530: /home/pi/.local/share/applications/mpv.desktop: No such file or directory
```

**Comment by CrashCortez** [comment]
This is in the notes section 

I Suggest you use the upcomming release, the MPV and FFMPEG tutorial can be hard to follow. I did automate it, to see if that will help make the set up eaiser. all credit goes to RPi_Mike and his killer script. Go to the notes section click the link and read the tutorial.
